bloom-housing/bloom
# Create units summary table

## Issue
https://github.com/CityOfDetroit/bloom/issues/238

Addresses # (issue)


## Description

Creates a UnitsSummary entity and dto, and the corresponding migration.

**Caveat**: I wasn't really sure how to structure the dto, would love some feedback there.

The table structure is described [here](https://github.com/bloom-housing/bloom/issues/1588#issuecomment-892034445), plus a `property` column which holds a `property.id` foreign key.

Current primary keys for the new table are `unitType`, `property`, and `monthlyRent`, but I expect we may want to add more fields to the composite key.

## Type of change


## How Can This Be Tested/Reviewed?

Run the migration and view the newly created table. I manually added a few rows to verify the primary keys.

## Checklist:
